src/applications/personalization/profile-2/routes.js

```diff
@@ -1,20 +1,8 @@
-import { lazy } from 'react';
```

Comment:
Using lazy was causing all the GET requests to fire the first time you went to a Profile section. If you returned to a section after leaving it, the GET calls did not occur again. I'm not sure exactly what's going on with that. But importing and using the components the old fashioned way works as expected and isn't causing any issues. @sanlouise do you want to look into making lazy work? Or write a ticket to reimplement lazy?

Reply:
I can definitely look into this! Good catch 👍